src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -66,9 +64,19 @@ public String platform(Model theModel) {
 	}
 	@GetMapping("/list")
 	public String listGames(Model theModel) {
-		List<GameList> theGamelist = gameList.findAll();
-		theModel.addAttribute("gamelist", theGamelist);
+		List<GameList> theGameList = gameList.findAll();
+		theModel.addAttribute("gamelist", theGameList);
 		return "list-GameList";
 	}
 	
+	@GetMapping("/nintendoList")
+	public String nintendoList(Model theModel) {
+		// Model Attribute for data binding
+		GameList theGameList = gameList.getConsole("Nintendo");
+		
+		theModel.addAttribute("gamelist", theGameList);
+		
+		return "list-nintendo";
+	}
+	
```

Comment:
Attempted to get the Nintendo specific list to show up when clicking on the Nintendo button from PlatformList